Sources/PackageDescription/PackageRequirement.swift

```diff
@@ -173,9 +182,13 @@ extension Package.Dependency {
     /// requirements before publishing a version of your package.
     @available(_PackageDescription, introduced: 5.6)
     public enum SourceControlRequirement {
+        /// A version based requirement.
```

Comment:
As mentioned in #4247, this is not the only version-based requirement. Maybe we should emphasize the "exactness".